### PR TITLE
HDDS-11119. Unnecessary UPDATE_VOLUME audit entry for DELETE_TENANT

### PR DESCRIPTION
**Pull request by sumitagrawl**
## What changes were proposed in this pull request?

removal of duplicate audit logging of UPDATE_VOLUME for delete tenant case. Already DELETE_TENANT audit log is present.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-11119

## How was this patch tested?

- existing test cases for impact

### CODE REVIEW

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/audit/OMAction.java [hunk 1]
```diff
@@ -39,7 +39,6 @@ public enum OMAction implements AuditAction {
   RENAME_KEYS,
   SET_OWNER,
   SET_QUOTA,
-  UPDATE_VOLUME,
```
Comment (kerneltime) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/audit/OMAction.java:42]: Best not to change an enum.

Comment (sumitagrawl) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/audit/OMAction.java]: Fixed.

**Review by kerneltime — APPROVED** [review]

**Review by ashishkumar50 — APPROVED** [review]
@sumitagrawl Thanks for the patch, LGTM.

**Review by devmadhuu — APPROVED** [review]
Thanks @sumitagrawl for improving the patch. Changes LGTM +1

**Comment by ashishkumar50** [comment]
@sumitagrawl Thanks for the patch, @kerneltime @devmadhuu for the review.

**Pull request merged**